rclcpp/minimal_service/main.cpp

```diff
@@ -25,7 +25,9 @@ void handle_service(
   const std::shared_ptr<AddTwoInts::Response> response)
 {
   (void)request_header;
-  printf("request: %" PRId64 " + %" PRId64 "\n", request->a, request->b);
+  RCLCPP_INFO(
+    rclcpp::get_logger("minimal_service"),
```

Comment:
for free-function callbacks we've been capturing the logger in a callback and passing it to the handler: https://github.com/ros2/demos/pull/191/commits/52dbdac6dd29bfaab4cf938264bf43c8ffba46a8

Reply:
I thought this was unnecessary complexity here for just passing a string around so I didnt do it on purpose.

Reply:
Which string so you mean? If it's the logger name then the purpose is to have it reflect remapping on the node name (which isn't possible today but they'd be set up for when it's possible)

Reply:
I meant the logger. I think that as it stands it doesnt deserve doubling the size of the main to propagate it. I integrated @wjwwood suggestion (#191) instead.

> If it's the logger name then the purpose is to have it reflect remapping on the node name

Yeah indeed that's important to leverage when this becomes available. I was just discussing this with wjwwood and we agreed that hopefully by then we will provide an easy way for user to pass additional argument to callbacks as logging from callback is something very common.